pennylane_lightning/core/src/simulators/lightning_qubit/gates/cpu_kernels/GateImplementationsLM.hpp

```diff
@@ -621,11 +625,11 @@ class GateImplementationsLM : public PauliGenerator<GateImplementationsLM> {
         } else {
             const std::size_t rev_wire = num_qubits - wires[0] - 1;
             const std::size_t rev_wire_shift = (one << rev_wire);
-            const auto [parity_high, parity_low] = revWireParity(rev_wire);
+            const auto parities = revWireParity(rev_wire);
```

Comment:
All good, though I wonder if it would be more maintainable to define explicit references called `parity_high` and `parity_low` to the terms, and use them instead of `first` and `second` throughout. WDYT?

Reply:
Yes, let's stick to this pattern. I'll just copy them to two const variables since they are `std::size_t`.